### ISSUE TITLE
Battle Mode in NERO Mod

### ISSUE THREAD

**Issue by GoogleCodeExporter** [nnrg/opennero#29]
```
Implement a Battle mode in the NERO mod that will allow for competetive play 
between trained teams.

Implement in /branches/nero then merge into trunk after completion.
```

Original issue reported on code.google.com by `AdamDz...@gmail.com` on 30 Aug 2010 at 2:22
- Blocked on: #43
